models/user.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
+    fName: {
+        type: String, 
+        required: [true, 'Please type in first name'],
+        trim: true
+    }, 
+    lName: {
+        type: String, 
+        required: [true, 'Please type in last name'],
+        trim: true
+    }, 
+    email: {
```

Comment:
Lets  add a light weight package to help validate the email format. You can do `npm i validator` Import it to the top of the page.

```
import validator from 'validator';
```

then in the `email` object 

```
validate: {
     validator: validator.isEmail,
     message: 'Please provide a valid email',
}
```

This will check the email is being written correct so `example@mail.com`. Make sure there an string with an @ then string then a .com or whatever they use. You can trim the this as well.